setup.sh

```diff
@@ -0,0 +1,289 @@
+helpstring="Usage: source $BASH_SOURCE [options]
```

Comment:
Why have you removed the shebang on this script?

Reply:
The script is supposed to be sourced, in which case the shebang has no effect. I added checks to prevent the user from running it (because of [this comment](https://github.com/cms-gem-daq-project/gem-plotting-tools/issues/96#issuecomment-392785774)).